### DISCUSSION TITLE
Grouping the table with a cell which has multiple values

### DISCUSSION THREAD

**Discussion by ghost** [TanStack/table#3036] [Q&A]
Hi There,

I have a scenario where one of my table columns contains multiple ENUMS 

**For Example:** [
{ userId: 1,  hobbies: [chess, sing, cook] },
{ userId: 2, hobbies: [chess, sing, cook] },
{ userId: 3, hobbies: [chess] },
{ userId: 4, hobbies: [sing] },

**As of now I get the results like:**
![image](https://user-images.githubusercontent.com/58845615/107194247-ce201a80-6a15-11eb-8b26-44a9f52aefbe.png)

1. [chess, sing, cook]
2. [chess]
3. [sing]

** I want the table which contains a sub-table as**
![image](https://user-images.githubusercontent.com/58845615/107194317-e42ddb00-6a15-11eb-97c7-df0f1a779a09.png)

1. sing: The Sub Table should have columns, userId, ...other Columns [1, 2, 4]
2. chess: should contain [1, 2, 3]
3. cook: should Contain [1, 2]

Thanks in advance.

**Comment by ReinaldoSupplyScope** [comment]
Any luck on this?
